coala_quickstart/Extensions.py

```diff
@@ -0,0 +1,172 @@
+exts = {'.w': 'C',
+        '.cs': 'C#',
+        '.csx': 'C#',
+        '.c': 'C++',
+        '.c++': 'C++',
+        '.cpp': 'C++',
+        '.cxx': 'C++',
+        '.h': 'C++',
+        '.h++': 'C++',
+        '.hh': 'C++',
+        '.hpp': 'C++',
+        '.hxx': 'C++',
+        '.tcc': 'C++',
+        '.tpp': 'C++',
+        '.cmake': 'CMake',
+        '.cmake.in': 'CMake',
+        '.css': 'CSS',
+        '._coffee': 'CoffeeScript',
+        '.coffee': 'CoffeeScript',
+        '.cson': 'CoffeeScript',
+        '.iced': 'CoffeeScript',
+        '.cu': 'Cuda',
+        '.cuh': 'Cuda',
+        '.dart': 'Dart',
+        '.f': 'FORTRAN',
+        '.f03': 'FORTRAN',
+        '.f08': 'FORTRAN',
+        '.f77': 'FORTRAN',
+        '.f90': 'FORTRAN',
+        '.f95': 'FORTRAN',
+        '.for': 'FORTRAN',
+        '.fpp': 'FORTRAN',
+        '.go': 'Go',
+        '.htm': 'HTML',
+        '.html': 'HTML',
+        '.xhtml': 'HTML',
+        '.hs': 'Haskell',
+        '.hsc': 'Haskell',
+        '.json': 'JSON',
+        '.sublime-keymap': 'JSON',
+        '.sublime-mousemap': 'JSON',
+        '.sublime-project': 'JSON',
+        '.sublime-settings': 'JSON',
+        '.sublime-workspace': 'JSON',
+        '.sublime_metrics': 'JSON',
+        '.sublime_session': 'JSON',
+        '.java': 'Java',
+        '._js': 'JavaScript',
+        '.bones': 'JavaScript',
+        '.jake': 'JavaScript',
+        '.js': 'JavaScript',
+        '.jsfl': 'JavaScript',
+        '.jsm': 'JavaScript',
+        '.jss': 'JavaScript',
+        '.jsx': 'JavaScript',
+        '.pac': 'JavaScript',
+        '.sjs': 'JavaScript',
+        '.ssjs': 'JavaScript',
+        '.jl': 'Julia',
+        '.lua': 'Lua',
+        '.nse': 'Lua',
+        '.rbxs': 'Lua',
+        '.markdown': 'Markdown',
+        '.md': 'Markdown',
+        '.mkd': 'Markdown',
+        '.mkdown': 'Markdown',
+        '.ron': 'Markdown',
+        '.matlab': 'Matlab',
+        '.m': 'Objective-C',
+        '.mm': 'Objective-C',
+        '.cl': 'OpenCL',
+        '.opencl': 'OpenCL',
```

Comment:
OpenCL is a framework, what are these endings for actually? some super fast image processing language?